mautic/mautic
# Last active as contact field

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Yes
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR added Last active date field as not removed contact field. Same logic like with attribution_date.
The enhancement allow use last_date with campaign contact field condition. Make sense with new campaign capabilities like https://github.com/mautic/mautic/pull/6132 

Then we can create campaign with condition on last_active  date and restart campaign until last_active is not less as some time.  For example If contact's last active date is greather like 4 hours  then send email/notification (abandoned cart)...


#### Steps to test this PR:
1.  Apply PR, run `php app/console doctrine:schema:migration`
2.  Then go to campaigns and create campaign based on contact field condition with Date last active field
3. Should works on same way like Attribution date

![image](https://user-images.githubusercontent.com/462477/42172896-b80d4bdc-7e1d-11e8-8cd1-0fb2641e32a9.png)
